docs/migration/migrate_8_0.asciidoc

```diff
@@ -259,6 +259,12 @@ URL that it derived from the actual server address and `xpack.security.public` s
 
 *Impact:* Any workflow that involved manually clearing generated bundles will have to be updated with the new path.
 
+[float]
+=== Legacy `optimize.*` settings are no longer supported
+*Details:*  The legacy optimizer has been removed and deprecations for unused `optimize.*` settings have been logged since 7.10. The following settings have been removed.
```

Comment:
```suggestion
*Details:*  The legacy optimizer has been removed and any `optimize.*` settings have been deprecated since 7.10. These settings have been removed as they are no longer in use.
```

Reply:
@lukeelmers Thanks for the suggestions!
Would you mind resubmitting your review please? The PR was still in draft but is open for review now 🙏